mongock-spring-v5/src/main/java/com/github/cloudyrock/spring/v5/MongockTestConfiguration.java

```diff
@@ -1,16 +0,0 @@
-package com.github.cloudyrock.spring.v5;
-
-import org.springframework.context.annotation.Import;
-
-import java.lang.annotation.Documented;
-import java.lang.annotation.ElementType;
-import java.lang.annotation.Retention;
-import java.lang.annotation.RetentionPolicy;
-import java.lang.annotation.Target;
-
-@Documented
-@Target(ElementType.TYPE)
-@Retention(RetentionPolicy.RUNTIME)
-@Import({MongockConfiguration.class, MongockTestContext.class})
-public @interface MongockTestConfiguration {
-}
```

Comment:
Why is this removed?

Reply:
It was unused. If we want to keep it, I believe it should be part of the test classpath.

Reply:
It's requires in some test scenario, when they want Mongock to be enabled but still want to call the changeSet directly. It's an edge case for a user, but still relevant. I rather keep it ;)